src/Scss/base/_responsive.scss

```diff
@@ -284,6 +289,37 @@ $largeDesktop: rem(1920);
   .user_reply_comment {
     padding-left: 0 !important;
   }
+
+  // Ticket Page
+  .tickets_details {
+    .table_footer {
+      justify-content: center !important;
+      align-items: center !important;
+    }
+  }
+
+  .tickets_search {
```

Comment:
why didn't you used flex mixin for this? and whats the need of important in every property.

Reply:
Inside a mixin, we can't use `!important` to override an existing property, so I use a normal property and apply `!important` to override the existing property.